.github/workflows/generate-website-docs.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/create-github-app-token@v1
+      - uses: actions/create-github-app-token@v2
```

Comment:
@maxammann is it possible that the `GH_ORG_APP_` vars below are no longer available? 

It's complaning about `app-id` being missing https://github.com/KittyCAD/modeling-app/actions/runs/15971065696/job/45042166862?pr=7572#step:3:8